prototypes/multi-module-prototype/examples/multi-module-example/ComposeUI.Example.WPFDataGrid/Views/DataGridView.xaml.cs

```diff
@@ -33,7 +33,7 @@ public partial class DataGridView : Window
     private readonly ILogger<DataGridView> _logger;
     private readonly IMessageRouter _messageRouter;
     private readonly ObservableCollection<SymbolModel> _symbols;
-    private readonly IProcessInfoCollector _processInfoCollector;
+    private readonly IProcessInfoHandler _processInfoCollector;
```

Comment:
nitpick: When you're sure, that everything works, don't forget to also rename the variable to _processInfoHandler